app/experimenter/kinto/tasks.py

```diff
@@ -78,54 +78,46 @@ def nimbus_check_kinto_push_queue():
     for application, collection in NimbusExperiment.KINTO_APPLICATION_COLLECTION.items():
         kinto_client = KintoClient(collection)
 
-        rejected_collection_data = kinto_client.get_rejected_collection_data()
-        if rejected_collection_data:
-            rejected_slug = kinto_client.get_rejected_record()
-            experiment = NimbusExperiment.objects.get(slug=rejected_slug)
-            if (
-                experiment.status == NimbusExperiment.Status.LIVE
-                and experiment.is_end_requested
-            ):
-                experiment.is_end_requested = False
-            else:
-                experiment.status = NimbusExperiment.Status.DRAFT
-
-            experiment.save()
-
-            generate_nimbus_changelog(
-                experiment,
-                get_kinto_user(),
-                message=f'Rejected: {rejected_collection_data["last_reviewer_comment"]}',
-            )
+        if kinto_client.has_pending_review():
+            continue
+        elif rejected_collection_data := kinto_client.get_rejected_collection_data():
+            handle_rejection.delay(collection, rejected_collection_data)
+        elif queued_launch_experiment := NimbusExperiment.objects.launch_queue(
+            application
+        ).first():
+            nimbus_push_experiment_to_kinto.delay(queued_launch_experiment.id)
+        elif queued_pause_experiment := NimbusExperiment.objects.pause_queue(
+            application
+        ).first():
+            nimbus_end_experiment_in_kinto.delay(queued_pause_experiment.id)
+        elif queued_end_experiment := NimbusExperiment.objects.end_queue(
```

Comment:
flake8 is not happy with this syntax ![Screen Shot 2021-03-12 at 3 27 03 PM](https://user-images.githubusercontent.com/6392049/110994875-7abc2780-8347-11eb-8f48-cb1c91364121.png)

Reply:
Huh that's funny, sometimes flake8 and black don't get along, black knows what to do so whatever black formats it to if flake8 is still complaining just disable the flake8 warning.